Xamarin.Forms.Core/Editor.cs

```diff
@@ -68,6 +68,7 @@ public Editor()
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
+		void IEditorController.SendCompleted() => SendCompleted();
```

Comment:
Happy to add it. It's just that those two members are so closely related. Would you like me to add a new line for all such members going forward or would you like to reconsider?